must-gather/README.md

```diff
@@ -0,0 +1,37 @@
+OCS must-gather
+=================
+
+`lvm-must-gather` is a tool built on top of [OpenShift must-gather](https://github.com/openshift/must-gather)
```

Comment:
The name of this tool is OCS must-gather or "lvm must gather"?

Reply:
This will be lvm-must-gather